pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -0,0 +1,74 @@
+package dev_table_tooling
+
+import (
+	"context"
+	"encoding/base64"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// Encapsulates the binary path(s) of a command allowed to execute
+// along with a strict list of arguments.
+type AllowedCommand struct {
```

Comment:
```suggestion
// AllowedCommand encapsulates the possible binary path(s) of a command allowed to execute
// along with a strict list of arguments.
type AllowedCommand struct {
```


I've never been sure if structs comments are supposed to start with the struct name (analogous to methods). https://tip.golang.org/doc/comment implies not, but does recommend slightly different phrasing. I'd reword slightly